storage_service/storage_service/settings/base.py

```diff
@@ -354,6 +354,12 @@ def get_env_variable(var_name):
 def is_true(env_str):
     return env_str.lower() in ['true', 'yes', 'on', '1']
 
+def to_int(env_str):
+    try:
+        return int(env_str)
+    except ValueError:
```

Comment:
If the environment variable is not present, this will receive a value of None, and `TypeError` will be raised. We should keep catching `ValueError` too as it will catch garbage strings.

Reply:
Okay thanks - my python ability (or lack of) shining through! I'll fix that and resubmit.